recipe/meta.yaml

```diff
@@ -1,48 +1,49 @@
 {% set name = "cadquery" %}
-{% set version = "1.2.5" %}
+{% set version = "2.2.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  - folder: dist
-    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-    sha256: b2ddd927cb81bc9ad2332692d4ab0b4cb36581543463b64bd2d388b3709ea0ae
-    patches:
-      - osx_freecad_path.patch
   - folder: src
-    url: https://github.com/dcowden/cadquery/archive/v{{ version }}.tar.gz
-    sha256: 75eb6f987aeaf3d86197353c53dc2808a78fa9f2ade651f5751e96aa14b23829
+    url: https://github.com/CadQuery/cadquery/archive/refs/tags/{{ version }}.tar.gz
+    sha256: 9a71a3bb1185228f62b14810fe078489f4af61e02d9e2c683e68626ff9903dbc
 
 build:
   number: 0
   noarch: python
-  script: cd dist && python -m pip install --no-deps --ignore-installed .
+  script: cd src && python -m pip install --no-deps --ignore-requires-python --ignore-installed .
 
 requirements:
   host:
-    - python >=3.6,<3.7
+    - python >=3.8,<=3.11
+    - setuptools
     - pip
   run:
-    - python >=3.6,<3.7
-    - pyparsing
-    - freecad
+    - python >=3.8,<=3.11
+    - ocp 7.7.*
+    - pyparsing >=2.1.9
+    - ezdxf
+    - ipython
+    - typing_extensions
+    - nptyping 2.0.1
```

Comment:
hard pins are... hard. guess it is what is, but all the more reason to `pip check` below

Reply:
I'll try with >=

Reply:
I wasn't arguing too hard about the hard pin, as that's what's used upstream, just that it can get hard in the future to match this for e.g. new platforms.

The point was rather that `pip check` below will help find when/if that number does change.

Reply:
I put the hard pin back, thinking it might be related to the `imath` fail, but this might be something deeper:

- https://github.com/conda-forge/gmsh-feedstock/issues/71
- https://github.com/conda-forge/imath-feedstock/issues/7

We might be in a bit of a holding pattern until the latter issue is resolved

Reply:
The `imath` failure is for sure not related to `nptyping`. I do want to get rid of the hard pin. The only reason that is a thing upstream is my putting it there without too much thought.

Reply:
Hooray. I'm just reading tea leaves.

Looks like a fix is in the offing: https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/413